### DISCUSSION TITLE
GS can't terraform/plant trees as deity

### DISCUSSION THREAD

**Discussion by andythenorth** [OpenTTD/OpenTTD#11703] [Features]
### Version of OpenTTD

13-ish

### Expected result

GS landscape-modifying actions are not restricted in deity mode.

### Actual result

GS in deity mode cannot perform certain actions, such as modifying tile height, planting trees etc.

Peter has a branch that fixes this for the cases I tried. 
https://github.com/PeterN/OpenTTD/commit/6734a6981cc70c13973ac0c5b3b179542fe6cd9c

- does it catch all cases? (Probably does?)
- needs PR
- I may need to provide a test GS?

### Steps to reproduce

Write a GS that tries to perform landscaping actions as deity.
Observe that actions are not performed.

**Comment by 2TallTyler** [comment]
I'd approve the PR mentioned in this issue. 🙂
